packages/config/src/layer2s/arbitrum.ts

```diff
@@ -26,13 +41,16 @@ export const arbitrum: Layer2 = {
     slug: 'arbitrum',
     warning:
       'Fraud proof system is fully deployed but is not yet permissionless as it requires Validators to be whitelisted.',
-    description:
-      "Arbitrum One is an Optimistic Rollup that aims to feel exactly like interacting with Ethereum, but with transactions costing a fraction of what they do on L1.\
+    description: `Arbitrum One is an Optimistic Rollup that aims to feel exactly like interacting with Ethereum, but with transactions costing a fraction of what they do on L1.\
       Centralized Sequencer receives users' transactions and regularly sends the transaction batch to mainnet Ethereum. Independent Validators (currently whitelisted)\
-      read transaction batches from L1, execute them and submit a resulting L2 state root to L1. Any other Validator can challenge the state root within the challenge window (7-days). \
+      read transaction batches from L1, execute them and submit a resulting L2 state root to L1. Any other Validator can challenge the state root within the challenge window (${formatSeconds(
+        challengeWindow * assumedBlockTime,
+      )}). \
       The challenge will result in an interactive fraud proof game that will be eventually settled by L1. As long as there is at least one honest Validator, users are guaranteed that\
       eventually correct L2 state root will be published to L1. If Sequencer is censoring users transactions, it is possible to force the transaction via L1 queue. If no Validator publishes\
-      L2 state root within 7 days, the Validator whitelist is dropped and anyone can take over as a new Validator.",
+    L2 state root within ${formatSeconds(
+      validatorAfkBlocks * assumedBlockTime,
+    )} (${validatorAfkBlocks} blocks), the Validator whitelist is dropped and anyone can take over as a new Validator.`,
```

Comment:
Maybe extracting `validatorAfkTime` to a variable would make this more readable.